vendor/bat-native-ledger/src/bat/ledger/internal/database/database_activity_info.cc

```diff
@@ -170,7 +171,23 @@ void DatabaseActivityInfo::NormalizeList(
       _1,
       callback);
 
-  ledger_->RunDBTransaction(std::move(transaction), transaction_callback);
+  auto shared_list = std::make_shared<ledger::PublisherInfoList>(
+      std::move(list));
+
+  ledger_->GetLedgerClient()->RunDBTransaction(
```

Comment:
Is `transaction_callback` no longer used?

Reply:
removed